// See README.txt for information and build instructions.
syntax = "proto2";

package msg;

message RegisterRequest {
	required  string user_name = 1;
	required  string password = 2;
}

message RegisterResponse {
	required  bytes result = 1;
	required   string token = 2;
}

message LoginRequest {
   required string user_name = 1;
   required string password = 2;
}

message LoginResponse {
   required bytes result = 1;
   required string token = 2;
}




